packages/lib/src/core/core.ts

```diff
@@ -80,6 +80,10 @@ class Core {
 
         this.createCoreModules();
 
```

Comment:
Could we then remove the call for `this.createPaymentMethodsList()` in the Core.constructor, and let it be done only in the `initialize()` function? 

It seems to make sense to me. Otherwise, when initializing the lib, we would be calling this twice on advanced and sessions flow

Reply:
It makes sense! Will update.